src/Collections/Upstreams.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pantheon\Terminus\Collections;
+
+use Terminus\Exceptions\TerminusNotFoundException;
+
+class Upstreams extends UserOwnedCollection
```

Comment:
Hmm... Seems like the URL doesn't require the user ID. In that case I'd say this is NOT being accessed through the user (even though it's presumably filtered by user access). That makes sense conceptually since we're asking 'what upstreams are available on the platform' not 'what upstreams does this user own'.

TL;DR I vote normal collection